test/parallel/test-debugger-extract-function-name.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const common = require('../common');
+
+common.skipIfInspectorDisabled();
+
+const fixtures = require('../common/fixtures');
+const startCLI = require('../common/debugger');
+
+const assert = require('assert');
+
+const cli = startCLI([fixtures.path('debugger', 'three-lines.js')]);
+
+function onFatal(error) {
+  cli.quit();
+  throw error;
+}
+
+cli.waitForInitialBreak()
+  .then(() => cli.waitForPrompt())
+  .then(() => cli.command('exec a = function func() {}; a;'))
+  .then(() => assert.match(cli.output, /\[Function: func\]/))
+  .then(() => cli.command('exec a = function func () {}; a;'))
+  .then(() => assert.match(cli.output, /\[Function\]/))
```

Comment:
The current logic seems to not consider white spaces between function name and brackets.